package.json

```diff
@@ -35,20 +35,21 @@
     "ember-cli-babel": "^6.10.0"
   },
   "devDependencies": {
-    "ember-cli": "^2.17.1",
     "ember-cli-dependency-checker": "^2.0.0",
     "ember-cli-eslint": "^4.2.1",
-    "ember-cli-htmlbars": "^2.0.1",
     "ember-cli-htmlbars-inline-precompile": "^1.0.0",
+    "ember-cli-htmlbars": "^2.0.1",
     "ember-cli-inject-live-reload": "^1.4.1",
     "ember-cli-pretender": "^1.0.1",
-    "ember-cli-test-loader": "^2.2.0",
     "ember-cli-shims": "^1.2.0",
+    "ember-cli-test-loader": "^2.2.0",
+    "ember-cli": "^2.17.1",
```

Comment:
I don't think that is the default order that npm and yarn use. I usually use https://github.com/keithamus/sort-package-json to sort this correctly.